test/include/utils/GrayscalePixel/Constants.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+#include <utils/GrayscalePixel/Types.hpp>
+
+extern const types::CharContainer<10> TEST_DATA = {
```

Comment:
Fix it then ship it: 

Static or give value in a .cpp
This will break linking if included in multiple .cpp files.